crypto/kzg/src/lib.rs

```diff
@@ -41,20 +46,31 @@ impl From<c_kzg::Error> for Error {
 #[derive(Debug)]
 pub struct Kzg {
     trusted_setup: KzgSettings,
+    context: PeerDASContext,
 }
 
 #[automock]
 impl Kzg {
     /// Load the kzg trusted setup parameters from a vec of G1 and G2 points.
     pub fn new_from_trusted_setup(trusted_setup: TrustedSetup) -> Result<Self, Error> {
+        // Initialize the trusted setup using default parameters
+        //
+        // Note: One can also use `from_json` to initialize it from the consensus-specs
+        // json string.
+        let peerdas_trusted_setup = PeerDASTrustedSetup::default();
```

Comment:
It gets compressed using libflate with a compression ratio of about 2, so its around 400KB on disk. We can put it under a feature flag if its an issue.

The `default` method name is because it implements the `default` trait. We could use a user defined method for clarity, though not a blocker I think since Default::default is pretty common